support/config/probes.go

```diff
@@ -17,6 +17,21 @@ func (p LivenessProbes) ApplyTo(podSpec *corev1.PodSpec) {
 
 func (p LivenessProbes) ApplyToContainer(container string, c *corev1.Container) {
 	if probe, ok := p[c.Name]; ok {
+		if probe.TimeoutSeconds == 0 {
+			probe.TimeoutSeconds = 30 // default is 1s, but we want to be more patient
+		}
+		if probe.PeriodSeconds == 0 {
+			probe.PeriodSeconds = 60 // default is 10s, but we want check liveness less frequently
+		}
+		if probe.InitialDelaySeconds == 0 {
+			probe.InitialDelaySeconds = 60 // allow time for pod to start and stabilize
+		}
+		if probe.FailureThreshold == 0 {
```

Comment:
Not sure if it's right but we might want to be stricter - feel free to reconfigure as a caller to a higher failure threshold, but never dip below 3?

Reply:
I was wanting to keep this a unbespoke as possible.  If the value is not set, the creator of the DeploymentConfig should expect it will be defaulted.  However, if it is explicitly set, but then changed, that is unexpected.  We do this for `initialDelaySeconds` on readiness probes, but that is because there is no rationale for it if the `period` is set reasonably.